tutor/__init__.py

```diff
@@ -1,8 +1,10 @@
 from flask import Flask  
 from tutor.config import DevelopmentConfig
 from flask_sqlalchemy import SQLAlchemy
+from flask_bcrypt import Bcrypt
 
 
 app = Flask(__name__) 
 app.config.from_object(DevelopmentConfig)
 db = SQLAlchemy(app)
+bcrypt = Bcrypt(app)
```

Comment:
I think it would be better to import it on the specific controllers, because not all the app need that package but only the register and login. But I'm not quite sure how if it really important, so maybe @ifireball can help.